vividus-plugin-web-app/src/main/resources/steps/defaults/scroll.steps

```diff
@@ -0,0 +1,11 @@
+Composite: When I scroll to the end of the page
+When I scroll context to BOTTOM edge
```

Comment:
risky change, there could be cases, when some context is set and user wants to scroll the page, but not context

Reply:
It is, but shouldn't we cut it as soon as possible?

Reply:
breaking changes are always painful for users, so I recommend to Deprecate these steps